packages/cli-platform-ios/src/index.ts

```diff
@@ -4,5 +4,11 @@
 
 export {default as commands} from './commands';
 export {projectConfig, dependencyConfig, findPodfilePaths} from './config';
+
 export {default as getArchitecture} from './tools/getArchitecture';
 export {default as installPods} from './tools/installPods';
+
+// TODO: Move this to `cli-platform-apple`
+export {commandBuilder as buildPlatformCommand} from './commands/buildIOS';
+export {commandBuilder as buildPlatformRunCommand} from './commands/runIOS';
+export {commandBuilder as buildPlatformLogCommand} from './commands/logIOS';
```

Comment:
I think it would be more readable if we have separate files called “createBuild”, “createRun”, “createLog” and existing functions would only import these helpers and configure for iOS platform